share/lib/python/neuron/gjtests/ggap.mod

```diff
@@ -0,0 +1,20 @@
+: ggap.mod
+: Conductance-based gap junction model
+
+NEURON {
+    POINT_PROCESS ggap
+    RANGE g, i, vgap
+    NONSPECIFIC_CURRENT i
```

Comment:
@iraikov  With your permission, I would like to replace the NONSPECIFIC_CURRENT i with ELECTRODE_CURRENT i so that the mod file works properly in the eventuality that the extracellular mechanism is used. i.e the voltage needs to be the internal voltage and not the transmembrane voltage when those are different. This also means I would replace the two occurrences of (v - vgap) with (vgap - v)

@pramod Given the need to make ggap.mod available to run the gjtests/test_par_gj.py, is it better to move this somewhere into nrn/tests  (do those tests allow mod files?)

Reply:
@nrnhines of course you have my permission, but the only purpose of this mechanism is to test the parallel transfer functionality. There is no need for it if there are more rigorous existing tests.

Reply:
> @pramod Given the need to make ggap.mod available to run the gjtests/test_par_gj.py, is it better to move this somewhere into nrn/tests (

yeah, it would be nice to have tests under top level directory `nrn/test`.